src/components/App.js

```diff
@@ -7,31 +7,34 @@ import { connect } from 'react-redux'
 import BetaBanner from './BetaBanner'
 import BetaModal from './BetaModal'
 import Disclaimer from './Disclaimer'
+import Feedback from './Feedback'
 import Footer from './Footer'
 import Glossary from './Glossary'
 import Header from './Header'
-import { hideModal } from '../actions/modal'
 
 const App = ({ appState, children, dispatch, location }) => (
   <div className="site">
     <Disclaimer />
-    <BetaBanner />
+    <BetaBanner dispatch={dispatch} />
```

Comment:
question for you:

instead of passing the `dispatch` to these new components, what if we simply passed the dispatch calls with the proper actions (so that the child components don't need to know about or import the various actions); i don't think this is the best is all cases maybe it's simpler when there's only 1 or 2 actions that a component can do; i'm not sure about this but wanted to get your thoughts